### PR TITLE
Add minutes!

### PR DESCRIPTION
**Pull request by damacus**
This change is [Reviewable](https://reviewable.io/reviews/sous-chefs/meta/70)

### CODE REVIEW

File: minutes/2017-05-08.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- foodcritic (external)
+
+## Projects website
+
+- sc-generator
+- sc-foodcritic-rules
+
+## Minutes:
+
+- redisio: should be making a release in the next day or too which a _LARGE_ amount of updates sc-mongodb
```
Comment (shortdudey123) [inline minutes/2017-05-08.md]: wrong cookbook at the end :) 
that might be my bad in the google doc though

File: minutes/2017-05-08.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Postgresql: still working on cluster testing in test-kitchen
```
Comment (shortdudey123) [inline minutes/2017-05-08.md]: `sc-mongodb`

File: minutes/2017-05-08.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- HAProxy: review of OS specific issues in progress HAProxy
```
Comment (shortdudey123) [inline minutes/2017-05-08.md]: `haproxy`

**Review by shortdudey123 — CHANGES_REQUESTED** [review]
several cookbook names got mixed up

**Comment by rshade** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sous-chefs/meta/70)*

**Pull request merged**